cgi-bin/ljlib.pl

```diff
@@ -39,6 +39,7 @@ BEGIN
 
 BEGIN {
     # mod_perl does this early too, make sure we do as well
+    $LJ::_T_RUNNING = 1 if $0 && $0 =~ m!^t/!;
```

Comment:
I think that's fine and is less likely to accidentally break in
the future.


On Wed, Jan 28, 2015, at 11:15 PM, Mark Smith wrote:
> In cgi-bin/ljlib.pl[1]:


> > @@ -39,6 +39,7 @@ BEGIN
>
>  BEGIN {
>      # mod_perl does this early too, make sure we do as well
> +    $LJ::_T_RUNNING = 1 if $0 && $0 =~ m!^t/!; Do you prefer this
>      over having something in each test file that says "I am a test,
>      you are in the test environment"?


> — Reply to this email directly or view it on GitHub[2].

>



Links:

  1. https://github.com/dreamwidth/dw-free/pull/931#discussion_r23750212
  2. https://github.com/dreamwidth/dw-free/pull/931/files#r23750212